lib/fs.js

```diff
@@ -1615,6 +1615,11 @@ function ReadStream(path, options) {
   if (!(this instanceof ReadStream))
     return new ReadStream(path, options);
 
+  if (typeof options === 'string')
```

Comment:
It's an additive change so I think just semver-minor will do?  I kind of like it, it lets you write more succinct code.